commit 826c7ee3f61bde713c9267d1817d8d906fb34dde (refs/changes/16/4616/2)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2018-08-10T13:29:53-04:00 (1 year, 2 months ago)
    
    OS-7009 varpd fails to start on boot, missing overlay module
    Reviewed by: Jason King <jbk@joyent.com>
    Reviewed by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/cmd/zoneadm/zones.xml b/usr/src/cmd/zoneadm/zones.xml
index 1b6f8cd49b..b094bc660b 100644
--- a/usr/src/cmd/zoneadm/zones.xml
+++ b/usr/src/cmd/zoneadm/zones.xml
@@ -54,6 +54,19 @@
 		<service_fmri value='svc:/milestone/multi-user-server' />
 	</dependency>
 
+	<!--
+	    Until overlay device creation is moved out of the zone
+	    state-change script, zones must be dependent on varpd's
+	    successful launch.
+	-->
+	<dependency
+		name='varpd'
+		type='service'
+		grouping='require_all'
+		restart_on='none'>
+		<service_fmri value='svc:/network/varpd' />
+	</dependency>
+
 	<dependency
 		name='metadata'
 		type='service'
